website/en/docs/types/utilities.md

```diff
@@ -145,6 +145,8 @@ type Obj = {
 type MappedObj = $ReadOnly<$ObjMap<Obj, TypeFn>> // Still read-only
 ```
 
+> Note: $ReadOnly is only for making read-only object types. See the Array docs for [how to type read-only arrays](../arrays/#toc-readonlyarray) with `$ReadOnlyArray`.
```

Comment:
Suggestions I didn’t think of before:

- `Note: $ReadOnly` should be ``Note: `$ReadOnly` ``, with backticks
- Emphasize the word “object”: `read-only _object_ types`

Comments on your added text:

- I think the link would sound better if its bounds were changed from “how to type read-only arrays” to “type read-only arrays with `$ReadOnlyArray`”:

> See the Array docs for how to [type read-only arrays with `$ReadOnlyArray`.](../arrays/#toc-readonlyarray)

This is more natural because `$ReadOnlyArray` is the only way to do it, not just an example of how to type read-only arrays.